src/import/curl-util.c

```diff
@@ -252,7 +252,11 @@ int curl_glue_make(CURL **ret, const char *url, void *userdata) {
         if (curl_easy_setopt(c, CURLOPT_LOW_SPEED_LIMIT, 30L) != CURLE_OK)
                 return -EIO;
 
+#if LIBCURL_VERSION_NUM >= 0x075500 // libcurl 7.85.0
+        if (curl_easy_setopt(c, CURLOPT_PROTOCOLS_STR, "http,https,file") != CURLE_OK)
```

Comment:
(hmm, in the example they don't; confusing... not sure what to do now. kinda would use uppercase since we apparently need to use uppercase when reading the SCHEME opt, as in the first commit)

Reply:
btw, please use `/* */` for explanatory comments like here. we use `//` only for fixme style stuff...

Reply:
ok, but let's at least stick to one casing everywhere, i.e. in both CURLOPT_PROTOCOLS_STR and CURLINFO_SCHEME

Reply:
Yeah, the whole documentation is... confusing, will fix.

Reply:
in fact, if they document this is is case insensitive, maybe accordingly, use STRCASE_IN_SET() in the CURLINFO_SCHEME case, rather than STR_IN_SET()